### PR TITLE
Fixed settings and editor collapse/expand animation blicking on first render

### PR DESCRIPTION
**Pull request by AlexanderBazukevich**
Updates:
 - animation styles for settings and editor moved to root and is set by corresponding attributes;
 - removed js code responsible for animation class toggling in settings and editor.

### CODE REVIEW

**Comment by AlexanderBazukevich** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/plugins/editor/editor.ts [hunk 1]
```diff
@@ -128,10 +128,9 @@ export class UIPEditor extends UIPPlugin {
...
-        return this.setEditorConfig({theme: value === null ?
-            UIPEditor.defaultOptions.theme : UIPEditor.themesMapping['uip-dark']});
-      case 'editor-collapsed':
-        return this.classList.toggle('collapsed', value !== null);
+        return this.setEditorConfig({theme: value === null
+          ? UIPEditor.defaultOptions.theme
+          : UIPEditor.themesMapping['uip-dark']});
       default:
         return;
     }
```
Comment (yadamskaya) [inline src/plugins/editor/editor.ts:136]: Single case looks a bit weird, better use

```
if (attr === 'dark-theme') {
      return this.setEditorConfig({theme: value === null 
          ? UIPEditor.defaultOptions.theme
          : UIPEditor.themesMapping['uip-dark']});
```

**Review by yadamskaya — APPROVED** [review]

**Review by nattallius — APPROVED** [review]

**Review by bhlynsky — APPROVED** [review]

File: src/plugins/editor/editor.ts (hunk 1 above)
Comment (AlexanderBazukevich) [reply src/plugins/editor/editor.ts]: Done

**Pull request merged**